doc/sphinx/README.template.rst

```diff
@@ -153,6 +179,86 @@ DON'T
 
      .. tacv:: assert form as intro_pattern
 
+Using the ``.. coqtop::`` directive for syntax highlighting
+-----------------------------------------------------------
+
+DO
+  .. code::
+
+     A tactic of the form:
+
+     .. coqdoc::
+
+        do [ t1 | … | tn ].
+
+     is equivalent to the standard Ltac expression:
+
+     .. coqdoc::
+
+        first [ t1 | … | tn ].
+
+DON'T
+  .. code::
+
+     A tactic of the form:
+
+     .. coqtop:: in
+
+        do [ t1 | … | tn ].
+
+     is equivalent to the standard Ltac expression:
+
+     .. coqtop:: in
+
+        first [ t1 | … | tn ].
+
+Overusing plain quotes
+----------------------
+
+DO
+  .. code::
+
+     The :tacn:`refine` tactic can raise the :exn:`Invalid argument` exception.
+     The term :g:`let a = 1 in a a` is ill-typed.
+
+DON'T
+  .. code::
+
+     The ``refine`` tactic can raise the ``Invalid argument`` exception.
+     The term ``let a = 1 in a a`` is ill-typed.
+
+Plain quotes produce plain text, without highlighting or cross-references.
+
+Overusing the ``example`` directive
+-----------------------------------
+
+DO
+  .. code::
+
+     Here is a useful axiom:
+
+     .. coqdoc::
+
+        Axiom proof_irrelevance : forall (P : Prop) (x y : P), x=y.
+
+DO
+  .. code::
+
+     .. example:: Using proof-irrelevance
+
+        If you assume the axiom above, …
+
+DON'T
+  .. code::
+
+     Here is a useful axiom:
+
+     .. example::
+
+        .. code::
```

Comment:
You probably meant `coqdoc` here and BTW this part is only in the template but not in the generated README.